stencil-library/src/api/SubAccount.ts

```diff
@@ -149,3 +160,68 @@ export class SubAccount implements ISubAccount {
     return typeMap[this.status];
   }
 }
+
+export interface SubAccountBusinessDetails {
+  url?: string,
```

Comment:
I did intentionally mark all as optional - the reason I did so was because the shape of the data coming back from the BE was so inconsistent that it became hard to assume that any given piece of data would be present. 

Maybe this is a case where it makes sense to make these into classes that implement these interfaces, just so that we can provide some type of value - 'Not Configured' or something and keep the shape of the DOM consistent by ensuring there is some type of value. 

Thoughts @jakemer 🤔

Reply:
I think it's fine for now, we should revisit when we need these types for something else though